.expeditor/verify.pipeline.yml

```diff
@@ -937,4 +937,4 @@ steps:
           os-version: "10.13"
           inherit-environment-vars: true
       buildkite:
-        timeout_in_minutes: 45
+        timeout_in_minutes: 60
```

Comment:
I had to make this exact change in on of my PRs yesterday.

I'm going to make a standalone PR for this now, and then we can all benefit :smile:

Reply:
Aaaand here we are: #7785

Reply:
Thanks @christophermaier!  I've removed this change.